test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please from the project requirement you should ignore your `test.md` file with the `gitignore` file